webapp/src/css/variables.less

```diff
@@ -116,6 +116,7 @@
 @sidebar-menu-desktop-width: 470px;
 @sidebar-menu-header-mobile: @filters-bar-mobile-height;
 @sidebar-menu-header-desktop: @filters-bar-desktop-height;
+@tab-nav-size: 80px;
```

Comment:
```suggestion
@tab-nav-width: 80px;
```

Reply:
It's used for height when it's mobile, and for width when it's desktop. That's why it was just `size` 😁